src/actors/symbolication.rs

```diff
@@ -894,3 +906,30 @@ fn symbolic_registers_to_protocol_registers(
         })
         .collect()
 }
+
+fn cficache_error_to_status(e: &CfiCacheError) -> ObjectFileStatus {
+    match e.kind() {
+        CfiCacheErrorKind::Fetching => ObjectFileStatus::FetchingFailed,
+        // nb: Timeouts during download are caught by Fetching
+        CfiCacheErrorKind::Timeout => ObjectFileStatus::TooLarge,
+        CfiCacheErrorKind::ObjectParsing => ObjectFileStatus::MalformedFile,
+
+        _ => {
+            capture_fail(e);
+            ObjectFileStatus::Other
+        }
+    }
+}
+
+fn symcache_error_to_status(e: &SymCacheError) -> ObjectFileStatus {
+    match e.kind() {
+        SymCacheErrorKind::Fetching => ObjectFileStatus::FetchingFailed,
+        // nb: Timeouts during download are caught by Fetching
+        SymCacheErrorKind::Timeout => ObjectFileStatus::TooLarge,
```

Comment:
Let's add an explicit `ObjectFileStatus::Timeout` status or similar to indicate that we timed out during processing. I think this might be quite valuable for us to distinguish from files that are simply too large. 

As an example: We might be downloading _very_ large debug files but only a subset of their information is usable line info. On the other hand, a file that's perfectly in our limits might take too long to process because of a regression in `symbolic` or some other bug.

Finally, I think it would make sense to capture these timeouts to Sentry, with extra info on which file it was. This will be extremely useful for debugging.

(same for CFI above)

Reply:
Effectively this means renaming `TooLarge` to `Timeout`, as we don't have size limits on symcaches. Is that correct?

Reply:
Correct 👍

Reply:
done